src/styles/common-styles.module.scss

```diff
@@ -1,69 +1,66 @@
-@import './theme.module.scss';
+@import './colors.scss';
 
 .intro-button {
     text-align: center;
-    font: $small-font;
-    background-color: $snow-white;
+    font: var(--paragraph2);
+    background-color: var(--snow-white);
     padding: 10px;
     width: 160px;
-    color: $charcoal-black;
+    color: var(--charcoal-black);
     border-radius: 30px;
     margin-top: 20px;
     cursor: pointer;
     &:hover {
         background-color: #b6c1f0;
-        color: $black;
+        color: var(--black);
     }
 }
 
 .primaryButton {
-    background-color: $black;
+    background-color: var(--black);
     display: flex;
     align-items: center;
-    font: $small-font;
-    color: $snow-white;
+    font: --var(--paragraph2);
+    color: var(--snow-white);
     border-radius: 35px;
     height: 42px;
     padding-left: 22px;
     padding-right: 22px;
-    border: 1px solid $charcoal-black;
+    border: 1px solid var(--charcoal-black);
     z-index: 1;
+    cursor: pointer;
+}
 
-    &:hover {
-        background-color: $charcoal-black;
-        cursor: pointer;
-    }
+.primaryButton:hover {
+    background-color: var(--white);
+    border: 1px solid var(--charcoal-black);	   
+    color: var(--charcoal-black);
 }
 
 .secondaryButton {
     height: 42px;
     display: flex;
     align-items: center;
-    font: $small-font;
+    font: var(--paragraph2);
     border: 1px solid transparent;
     cursor: pointer;
     padding-right: 22px;
     padding-left: 22px;
-    border: 1px solid white;
+    border: 1px solid var(--white);
     z-index: 1;
 }
```

Comment:
Not code related, but no border (transparent) and no interactivity (hover state) in this button. Don't know if thats what you want or not 🙃

Reply:
will double check with Shiree tomorrow :)